### PR TITLE
Ford: driver torque limit safety

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

File: board/safety/safety_ford.h [hunk 1]
```diff
@@ -2,6 +2,8 @@
 #define MSG_EngBrakeData          0x165   // RX from PCM, for driver brake pedal and cruise state
 #define MSG_EngVehicleSpThrottle  0x204   // RX from PCM, for driver throttle input
 #define MSG_DesiredTorqBrk        0x213   // RX from ABS, for standstill state
+#define MSG_BrakeSysFeatures      0x415   // RX from ABS, for vehicle speed
+#define MSG_Yaw_Data_FD1          0x91    // RX from GWM, for yaw rate
```
Comment (incognitojam) [inline board/safety/safety_ford.h:6]: originating module is RCM (restraints control module)

File: board/safety/safety_ford.h [hunk 2]
```diff
@@ -116,6 +116,22 @@ static bool ford_lkas_msg_check(int addr) {
       || (addr == MSG_IPMA_Data);
 }
 
+// Curvature rate limits
+const SteeringLimits FORD_STEERING_LIMITS = {
+  .angle_deg_to_can = 50000,  // 1 / (2e-5) rad to can
+  .angle_rate_up_lookup = {
+    {5., 25., 25.},
```
Comment (incognitojam) [inline board/safety/safety_ford.h:123]: should there only be two breakpoints like OP?

Comment (sshane) [reply board/safety/safety_ford.h]: `interpolate` does NOT detect the length properly lol, above 25 the output is 0, so blocked msgs

**Comment by sshane** [comment]
Moved to https://github.com/commaai/panda/pull/1353

**Pull request closed without merging**